controllers/mover/syncthing/syncthing_test.go

```diff
@@ -474,7 +477,62 @@ var _ = Describe("When an RS specifies Syncthing", func() {
 		})
 
 		Context("Syncthing API is being used properly", func() {
-			// TODO: test syncthing API
+			When("Syncthing server does not exist", func() {
+				It("Doesn't work", func() {
+					res, err := mover.Synchronize(ctx)
+					Expect(err).ToNot(BeNil())
+					Expect(res).To(Equal(cMover.InProgress()))
+				})
```

Comment:
This is a good place to test that the individual Syncthing API methods also fail when the Server is not provided.